app/frontend/src/components/CommunityGuidelines/CommunityGuidelines.tsx

```diff
@@ -83,22 +103,21 @@ export default function CommunityGuidelines({
         {error && <Alert severity="error">{error}</Alert>}
 
         <div className={classes.grid}>
-          {COMMUNITY_GUIDELINES.map((_, index) => (
+          {data.communityGuidelinesList.map((guideline, index) => (
             <React.Fragment key={index}>
               <Avatar className={classes.avatar}>
-                <SvgIcon
-                  component={COMMUNITY_GUIDELINE_ICONS[index]}
-                  fontSize="large"
-                  className={classes.icon}
-                />
+                <SvgIcon fontSize="large" className={classes.icon}>
+                  <svg
+                    dangerouslySetInnerHTML={{ __html: guideline.iconSvg }}
```

Comment:
Does this make  `<svg><svg>...`?

Reply:
I'm not sure. I had some issues getting this to work, if you know what the right way to do this is, please let me know!